core-java-modules/core-java-nio-2/src/test/java/com/baeldung/lock/FileLocksUnitTest.java

```diff
@@ -0,0 +1,48 @@
+package com.baeldung.lock;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.IOException;
+import java.nio.channels.FileLock;
+import java.nio.channels.NonReadableChannelException;
+import java.nio.channels.NonWritableChannelException;
+
+import org.junit.jupiter.api.Test;
+
+class FileLocksUnitTest {
+
+    @Test void givenAnInputStream_whenGetWriteLock_throwNonWritableChannelException() {
```

Comment:
The formatting still looks to be off?

Reply:
Can you please tell me what is off? I used the intelliJ settings I found in https://github.com/eugenp/tutorials/blob/master/intelliJ/intelliJ-formatter.xml